### ISSUE TITLE
JupyterLab extension not getting installed correctly

### ISSUE THREAD

**Issue by vinayvaddiparthi** [aws-samples/sagemaker-run-notebook#66]
The JupyterLab extension isn't getting installed correctly using the provided lifecycle configuration hooks in both Notebook instances and Studio.

Part of the reason seems to be due to the fact that it is referencing an old repo (`"https://github.com/awslabs/sagemaker_run_notebook.git`) [here](https://github.com/aws-samples/sagemaker-run-notebook/blob/30bb6ecb33855363170dd5d4b0a1e689753bd24c/labextension/package.json#L25) which doesn't exist anymore.

Should it be replaced with the current repo? Is any one using this extension?

**Comment by morales-michelle** [comment]
I am also unable to get this to work. I tried the [script ](https://github.com/aws-samples/sagemaker-run-notebook/blob/master/scripts/lifecycle-configuration/studio/install-run-notebook.sh). 

@vinayvaddiparthi did you have any luck with this? Or anyone else.

**Comment by mayan-aakash** [comment]
Facing the same issue. The extension doesn't show up anymore. I've tried changing my default Jupyter lab versions and running the lifecycle configuration script on app start. Any luck with this problem?

**Comment by chetapgit** [comment]
same here, not able to get it working

**Comment by vinayvaddiparthi** [comment]
I got this working by updating the Git URLs in the code to this repo. I have created a [PR](https://github.com/aws-samples/sagemaker-run-notebook/pull/67) with the required changes.

I also have a few releases in my [personal repo](https://github.com/vinayvaddiparthi/sagemaker-run-notebook/releases) with these changes which are working for me. Feel free to use that or build your own release and use that until the issue in this repo is fixed.

Happy to help with any questions that anyone might have.

**Comment by tomfaulhaber** [comment]
I don't think that those URL changes would fix these problems. While you are correct that those are the wrong URLs, I believe they are only used to document the package, not as part of the installation.

**Comment by Almenon** [comment]
The extension shows up for me, although I looked at the code and I don't know how it does it. The [start.sh](https://github.com/aws-samples/sagemaker-run-notebook/blob/master/scripts/lifecycle-configuration/notebook-instances/start.sh) script defines a EXTENSION_NAME but never references it and never runs the install command!

It seems to be like it should be doing something like this: https://github.com/aws-samples/amazon-sagemaker-notebook-instance-lifecycle-config-samples/blob/master/scripts/install-lab-extension/on-start.sh#L15 🤔

**Comment by vinayvaddiparthi** [comment]
Good to know that it is working fine for you!

My hunch is that [these](https://github.com/aws-samples/sagemaker-run-notebook/blob/30bb6ecb33855363170dd5d4b0a1e689753bd24c/labextension/package.json#L28-L32) commands build and install the extension and [fixing](https://github.com/aws-samples/sagemaker-run-notebook/pull/67) the repo URL fixed it. I'll close this issue since it is no longer an issue now.
